concordium-std/src/traits.rs

```diff
@@ -327,12 +327,12 @@ pub trait HasHost<State>: Sized {
         amount: Amount,
     ) -> ReadOnlyCallContractResult<Self::ReturnValueType>;
 
-    /// Like [`invoke_contract_raw`](Self::invoke_contract_raw), except that the
-    /// parameter is automatically serialized. If the parameter already
-    /// implements [`AsRef<[u8]>`](AsRef) or can be equivalently cheaply
-    /// converted to a byte array, then
-    /// [`invoke_contract_raw`](Self::invoke_contract_raw) should be
-    /// used, since it avoids intermediate allocations.
+    /// Like [`invoke_contract_raw_read_only`](Self::
+    /// invoke_contract_raw_read_only), except that the parameter is
```

Comment:
This doesn't work and leads to broken links in the generated documentation.
![Screenshot from 2023-01-06 18-27-15](https://user-images.githubusercontent.com/105212/211070325-22934c73-ee1d-4eee-aeec-f6a6d3143350.png)


Please be mindful of these issues.